packages/trader/src/Modules/Trading/Components/Form/screen-small.tsx

```diff
@@ -83,7 +86,13 @@ const CollapsibleTradeParams = ({
     const isVisible = (component: string) => form_components.includes(component);
 
     return (
-        <Collapsible position='top' is_collapsed={is_collapsed} onClick={onClick} handle_button>
+        <Collapsible
+            position='top'
+            is_collapsed={is_collapsed}
+            onClick={onClick}
+            handle_button
+            show_collapsible_button={TRADE_TYPES.EVEN_ODD === contract_type}
```

Comment:
<img width="867" alt="Screenshot 2024-02-19 at 12 48 54 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/64ee1f0c-bbc1-4342-bff1-802f8b9b7af9"> 
The 'handle' button has no effect here because this trade type is not supposed to be collapsible, nothing is getting collapsed or expanded in it, and there is no handle (collapsible wrapper) in production. This implementation will make users think that the  'handle' button  is broken. UI should be intuitive.

Please try using CSS approach for fixing this unless we have a design for this non-interactive collapsible wrapper. This note is there but it's simply hidden :) :
![image](https://github.com/binary-com/deriv-app/assets/103177211/7f452e23-0b8c-483f-8f41-168f201f2d80)

Reply:
Thank you for suggestion! In order to show, that trade params for Even/Odd are not collapsible, I removed handle button for this particular contract by adding conditional rendering. Because its' small incapsulated changes, it should not break other places, where the component is used.
<img width="680" alt="Screenshot 2024-02-19 at 14 42 51" src="https://github.com/binary-com/deriv-app/assets/121025168/f67067f3-68f3-4a00-8916-b778d1f795ac">